### PR TITLE
Fix double brackets

### LINKED ISSUE #10522: Double brackets in footnote references (docutils 0.18?)
### Describe the bug

See https://github.com/sphinx-doc/sphinx/issues/10415#issuecomment-1146642591, that issue also has screenshots.

### How to Reproduce

x

### Expected behavior

_No response_

### Your project

x

### Screenshots

_No response_

### OS

x

### Python version

x

### Sphinx version

x

### Sphinx extensions

_No response_

### Extra tools

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by AA-Turner**
Fixes #10522.

### Feature or Bugfix
- Bugfix

A

### CODE REVIEW

File: doc/templating.rst [hunk 1]
```diff
@@ -383,6 +383,15 @@ in the future.
 
    .. versionadded:: 4.2
 
+.. data:: docutils_version_info
+
+   The version of Docutils used to build represented as a tuple of five elements.
+   For Docutils version 0.16.1 beta 2 this would be `(0, 16, 1, 'beta', 1)``.
```
Comment (mgeier) [inline doc/templating.rst]:
```suggestion
   For Docutils version 0.16.1 beta 2 this would be `(0, 16, 1, 'beta', 2)``.
```

**Comment by mgeier** [comment]
Looks good: https://sphinx--10523.org.readthedocs.build/en/10523/usage/advanced/intl.html#sphinx-internationalization-details

File: CHANGES [hunk 2]
```diff
@@ -13,10 +13,15 @@ Deprecated
...
 
+* #10523: HTML Theme: Expose the Docutils's version info tuple as a template
+  variable, ``docutils_version_info``. Patch by Adam Turner.
+
 Bugs fixed
 ----------
 
 * #10509: autosummary: autosummary fails with a shared library
+* #10523: HTML Theme: Fix double brackets on citation references in Docutils 0.18+.
+  Patch by Adam Turner.
```
Comment (tk0miya) [inline CHANGES:25]: Is this a new format of CHANGES?

**Review by tk0miya — APPROVED** [review]
LGTM with nits.

File: CHANGES (hunk 2 above)
Comment (AA-Turner) [reply CHANGES:25]: I prefer capitalising HTML at the least, as it is an initialism -- I could leave "theme" lowercased?

A

**Comment by AA-Turner** [comment]
Merged `5.0.x` to fix conflicts.

A

File: CHANGES (hunk 2 above)
Comment (tk0miya) [reply CHANGES:25]: No, I did not care about that. My interest is "Patch by" notation. Do we have to append "Patch by who" to each CHANGES entry?

Comment (AA-Turner) [reply CHANGES:25]: > Do we have to append "Patch by who" to each CHANGES entry?

I'd say it is personal preference not absolutley not a requirement -- I prefer to put "Patch by Adam Turner". For example in CPython some people do and some don't (https://github.com/python/cpython/blob/main/Misc/NEWS.d/3.11.0b1.rst?plain=1).

A

Comment (tk0miya) [reply CHANGES:25]: Okay, understandable.

**Comment by tk0miya** [comment]
Thank you for your quick work :-)

**Pull request merged**